### PR TITLE
ENT-3247: Remove old LDAP implementation.

### PR DESCRIPTION
**Pull request by craigcomstock**

### CODE REVIEW

File: enterprise-cfengine-guide/hub_administration/custom-ldap-port.markdown [hunk 1]
```diff
@@ -15,33 +15,36 @@ via the Status and Setting REST API.
...
 
 ```console
 [root@hub ~]# export CACERT="/var/cfengine/httpd/ssl/certs/hub.cert"
-[root@hub ~]# export API="https://hub/api/settings"
-[root@hub ~]# export CURL="curl --netrc --silent --cacert ${CACERT} ${API}" 
-[root@hub ~]# ${CURL} | jq '.data[0] + {"ldapPort": 3268, "ldapPortSSL": 3269}' | ${CURL} -X POST -d @-
-[root@hub ~]# $CURL | jq '.data[0]'
+[root@hub ~]# export API="https://hub/ldap/settings"
+[root@hub ~]# export AUTH_HEADER="Authorization:<accessToken from settings.php as mentioned above>"
+[root@hub ~]# export CURL="curl --silent --cacert ${CACERT} -H ${AUTH_HEADER} ${API}"
```
Comment (nickanderson) [inline enterprise-cfengine-guide/hub_administration/custom-ldap-port.markdown]: Trailing whitespace nitpicking

**Comment by craigcomstock** [comment]
Thanks Nick! I'll fixup the whitespace.

**Comment by nickanderson** [comment]
OK merge when ready or just tell me and ill do it.

**Comment by craigcomstock** [comment]
we can coordinate all of the PRs I suppose. not that it's super critical.

**Comment by craigcomstock** [comment]
@nickanderson if this looks good to you, I am ready for you to merge it.

**Pull request merged**